ui/components/extensions/adapters/adapters.js

```diff
@@ -153,15 +169,6 @@ const Adapters = ({ updateProgress, classes, meshAdapters, meshAdaptersts }) =>
   )
 }
 
-const mapStateToProps = (state) => {
-  const meshAdapters = state.get("meshAdapters");
-  const meshAdaptersts = state.get("meshAdaptersts");
-
-  return {
-    meshAdapters,
-    meshAdaptersts
-  };
-};
```

Comment:
Can you please confirm, why we would have to use an extra fetch instead of relying on our Redux Store variables for consistency?

Reply:
If there is a lag between the values not being properly propagated when this UI components is loaded then we should probably check the cause for that bottleneck instead of switching over from the Redux store var to an extra GET request, IMO 🤔

WDYT?

Reply:
Yes there is lag. And then also issue of showing a loader. Because I am not sure how you would show the loader incase you are using a global state. I mean previously I did tried it but it resulted in more weird cases (e.g, a very short and abrupt loader). 
Since a loader should be related to the api request anyways.  Not so in this case though.

Reply:
A use case for an upgrade to redux-toolkit (now), @nebula-aac?